app/models/sample.rb

```diff
@@ -11,4 +11,12 @@ class Sample < ApplicationRecord
 
   validates :name, presence: true, length: { minimum: 3, maximum: 255 }
   validates :name, uniqueness: { scope: %i[name project_id] }
+
+  def self.ransackable_attributes(_auth_object = nil)
+    %w[id name created_at updated_at]
+  end
+
+  def self.ransackable_associations(_auth_object = nil)
+    %w[]
```

Comment:
Let me know if this is the fix you were expecting 8a63eba1bdc44cde36dfc1a38879ebc650c52fc8

Ransack suggests using the `authorizable_{method_name}` rather than super as it looks like the use of super in this  context will be deprecated and potentially cause errors in the future: `Ransack's builtin #{method} method is deprecated and will result
              in an error in the future. If you want to authorize the full list
              of searchable #{list_type} for this model, use
              authorizable_#{method} instead of delegating to super.`

Reply:
As discussed on teams, I reverted the changes back to how they were as we'll be implementing use of associations soon enough.